### ISSUE TITLE
Clarify certifier needs to be associated with conformance statement

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#1789]
We corrected that reports need to be associated with the certifier in #1410, but we don't indicate who has certified the conformance claim. We should similarly show an example with the certifier linked:

```
<meta property="dcterms:conformsTo" id="a11y-conf">EPUB-A11Y-11_WCAG-21-AA</meta>
<meta property="a11y:certifiedBy" refines="#a11y-conf">Acme Certifiers</meta>
```

The problem is we'll introduce warnings/errors if we require this normatively, so it has to only be an example.

I wonder if it would make sense to say refining is optional if only one conformance claim/certifier is included in the metadata, but otherwise should be used? (But also keep this an informative statement.)

It is needlessly cumbersome unless there are multiple conformance claims or certifiers.

**Comment by clapierre** [comment]
I agree this should be optional and strongly suggested if there are multiple conformsTo statements that the publisher includes in their OPF file.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-09-09-epub-a11y#section2) on 2021-09-09

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

### 2. Clarify certifier needs to be associated with conformance statement (pr epub-specs#1789)
{: #section2}

_See github pull request [#1789](https://github.com/w3c/epub-specs/issues/1789)._





**Matt Garrish:** this was a problem with tying conformance statement to certifier  
… this restores the refines attribute so that you can explicitly make that link between those metadata  
… at the time we had removed this on the expectation that refines was going to be gone  

</details>
